When doing HTTP Redirect Binding for saml SLO getting exception

**Describe the issue:**
[tenant mode]When doing HTTP Redirect Binding for saml SLO using the doc [1] getting exception

[1] https://is.docs.wso2.com/en/5.11.0/learn/handling-saml-single-logout-requests-from-federated-identity-providers/

**How to reproduce:**
1. Configure HTTP Redirect Binding with SAML SLO
2. When try to logout getting below exception
```
[2020-10-21 12:20:03,460] [41489392-af75-485c-a4d8-50df8708dfe4] ERROR {org.apache.catalina.core.ContainerBase.[Catalina].[localhost].[/].[bridgeservlet]} - Servlet.service() for servlet [bridgeservlet] in context with path [/] threw exception org.wso2.carbon.identity.application.authentication.framework.inbound.FrameworkRuntimeException: java.lang.NullPointerException
	at org.wso2.carbon.identity.application.authentication.framework.inbound.FrameworkRuntimeException.error(FrameworkRuntimeException.java:38)
	at org.wso2.carbon.identity.application.authentication.framework.inbound.IdentityServlet.getIdentityResponseFactory(IdentityServlet.java:216)
	at org.wso2.carbon.identity.application.authentication.framework.inbound.IdentityServlet.process(IdentityServlet.java:102)
	at org.wso2.carbon.identity.application.authentication.framework.inbound.IdentityServlet.service(IdentityServlet.java:44)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.eclipse.equinox.http.helper.ContextPathServletAdaptor.service(ContextPathServletAdaptor.java:37)
	at org.eclipse.equinox.http.servlet.internal.ServletRegistration.service(ServletRegistration.java:61)
	at org.eclipse.equinox.http.servlet.internal.ProxyServlet.processAlias(ProxyServlet.java:128)
	at org.eclipse.equinox.http.servlet.internal.ProxyServlet.service(ProxyServlet.java:68)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.wso2.carbon.tomcat.ext.servlet.DelegationServlet.service(DelegationServlet.java:68)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.owasp.csrfguard.CsrfGuardFilter.doFilter(CsrfGuardFilter.java:88)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.filters.HttpHeaderSecurityFilter.doFilter(HttpHeaderSecurityFilter.java:126)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.wso2.carbon.tomcat.ext.filter.CharacterSetFilter.doFilter(CharacterSetFilter.java:65)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.filters.HttpHeaderSecurityFilter.doFilter(HttpHeaderSecurityFilter.java:126)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:541)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.wso2.carbon.identity.cors.valve.CORSValve.invoke(CORSValve.java:89)
	at org.wso2.carbon.identity.context.rewrite.valve.TenantContextRewriteValve.invoke(TenantContextRewriteValve.java:105)
	at org.wso2.carbon.identity.authz.valve.AuthorizationValve.invoke(AuthorizationValve.java:110)
	at org.wso2.carbon.identity.auth.valve.AuthenticationValve.invoke(AuthenticationValve.java:101)
	at org.wso2.carbon.tomcat.ext.valves.CompositeValve.continueInvocation(CompositeValve.java:99)
	at org.wso2.carbon.tomcat.ext.valves.TomcatValveContainer.invokeValves(TomcatValveContainer.java:49)
	at org.wso2.carbon.tomcat.ext.valves.CompositeValve.invoke(CompositeValve.java:62)
	at org.wso2.carbon.tomcat.ext.valves.CarbonStuckThreadDetectionValve.invoke(CarbonStuckThreadDetectionValve.java:145)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690)
	at org.wso2.carbon.tomcat.ext.valves.CarbonContextCreatorValve.invoke(CarbonContextCreatorValve.java:57)
	at org.wso2.carbon.tomcat.ext.valves.RequestEncodingValve.invoke(RequestEncodingValve.java:49)
	at org.wso2.carbon.tomcat.ext.valves.RequestCorrelationIdValve.invoke(RequestCorrelationIdValve.java:126)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:373)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:868)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1590)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NullPointerException
	at org.wso2.carbon.identity.application.authenticator.samlsso.logout.util.SAMLLogoutUtil.getFederatedIdPConfigs(SAMLLogoutUtil.java:155)
	at org.wso2.carbon.identity.application.authenticator.samlsso.logout.processor.SAMLLogoutRequestProcessor.populateContextWithSessionDetails(SAMLLogoutRequestProcessor.java:223)
	at org.wso2.carbon.identity.application.authenticator.samlsso.logout.processor.SAMLLogoutRequestProcessor.process(SAMLLogoutRequestProcessor.java:119)
	at org.wso2.carbon.identity.application.authenticator.samlsso.logout.processor.SAMLLogoutRequestProcessor.process(SAMLLogoutRequestProcessor.java:74)
	at org.wso2.carbon.identity.application.authentication.framework.inbound.IdentityProcessCoordinator.process(IdentityProcessCoordinator.java:41)
	at org.wso2.carbon.identity.application.authentication.framework.inbound.IdentityServlet.process(IdentityServlet.java:84)
	... 53 more
```

**Expected behavior:**
Successfull logout

**Environment information** (_Please complete the following information; remove  any unnecessary fields_) **:**
  - Product Version:IS-5.11.0-beta2
 - OS: Linux
 - Database: POSTGRE
 - Userstore: JDBC

---

### Optional Fields

**Related issues:**
<!-- Any related issues from this/other repositories-->

**Suggested labels:**
<!-- Only to be used by non-members -->

--- a/components/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/logout/util/SAMLLogoutUtil.java
+++ b/components/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/logout/util/SAMLLogoutUtil.java
@@ -55,6 +55,7 @@ import java.security.cert.CertificateException;
 import java.security.cert.CertificateFactory;
 import java.security.cert.X509Certificate;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
@@ -152,13 +153,16 @@ public class SAMLLogoutUtil {
      */
     public static Map<String, String> getFederatedIdPConfigs(IdentityProvider identityProvider) {
 
-        Property[] properties = identityProvider.getDefaultAuthenticatorConfig().getProperties();
-        List<String> idpPropertyNames = Arrays.asList(SP_ENTITY_ID, SSO_URL, IS_AUTHN_RESP_SIGNED,
-                INCLUDE_CERT, IS_LOGOUT_REQ_SIGNED, IS_SLO_REQUEST_ACCEPTED);
-
-        return Arrays.stream(properties)
-                .filter(t -> idpPropertyNames.contains(t.getName()))
-                .collect(Collectors.toMap(Property::getName, Property::getValue));
+        List<String> idpPropertyNames = Arrays.asList(SP_ENTITY_ID, SSO_URL, IS_AUTHN_RESP_SIGNED, INCLUDE_CERT,
+                IS_LOGOUT_REQ_SIGNED, IS_SLO_REQUEST_ACCEPTED);
+        if (identityProvider.getDefaultAuthenticatorConfig() != null &&
+                identityProvider.getDefaultAuthenticatorConfig().getProperties() != null) {
+            Property[] properties = identityProvider.getDefaultAuthenticatorConfig().getProperties();
+            return Arrays.stream(properties)
+                    .filter(t -> idpPropertyNames.contains(t.getName()))
+                    .collect(Collectors.toMap(Property::getName, Property::getValue));
+        }
+        return Collections.emptyMap();
     }
 
     /**

